File: content/LLD/Reference-Code/1. OOPS Recap/1. Revision/classes_objects.py
class Student:
    # Methods
    def __init__(self, name: str, age: int, gender: str) -> None:
        # Attributes
        self.name = name
        self.age = age
        self.gender = gender

    def display(self) -> None:
        print(f"My name is {self.name}, age is {self.age} and gender is {self.gender}")

    def get_age(self) -> int:
        return self.age


s1 = Student("Anirudh", 22, "Male")
print(s1.get_age())

File: content/LLD/Reference-Code/1. OOPS Recap/1. Revision/ClassesObjects.java
class Student {
    // Attributes
    private String name;
    private int age;
    private String gender;

    // Methods
    public Student(String name, int age, String gender) {
        this.name = name;
        this.age = age;
        this.gender = gender;
    }

    public void display() {
        System.out.println("My name is " + this.name + ", age is " + this.age + " and gender is " + this.gender);
    }

    public int getAge() {
        return this.age;
    }
}

public class ClassesObjects {
    public static void main(String[] args) {
        Student s1 = new Student("Anirudh", 22, "Male");
        System.out.println(s1.getAge());
    }
}
